test/classes/deinitializers/deinitAndDestruct.good

```diff
@@ -0,0 +1,3 @@
+deinitAndDestruct.chpl:11: error: ambiguous call 'C.deinit()'
+deinitAndDestruct.chpl:2: note: candidates are: C.~C()
+deinitAndDestruct.chpl:6: note:                 C.deinit()
```

Comment:
I am curious what your thoughts were on whether we should always error if both a destructor and a deinitializer are present on the same type.  I'm asking because I figured you saw the discussion on the other PR, but no one else commented on it so I wasn't sure what position to assume everyone took.